snmp/datadog_checks/snmp/data/default_profiles/tp-link.yaml

```diff
@@ -0,0 +1,38 @@
+extends:
+  - _base.yaml
+  - _generic-if.yaml
+
+metadata:
+  device:
+    fields:
+      vendor:
+        value: "tp_link"
+
+sysobjectid:
+  - 1.3.6.1.4.1.11863.1.*  # TP-Link switch
+  - 1.3.6.1.4.1.11863.5.*  # TP-Link products
+metrics:
+  - MIB: TPLINK-SYSMONITOR-MIB
+    table:
+      name: tpSysMonitorCpuTable
+      OID: 1.3.6.1.4.1.11863.6.4.1.1.1
+    symbols:
+      - name: cpu.usage
+        OID: 1.3.6.1.4.1.11863.6.4.1.1.1.1.3
+    metric_tags:
+      - column:
+          OID: 1.3.6.1.4.1.11863.6.4.1.1.1.1.4
+          name: tpSysMonitorCpu5Minutes
+        tag: tp_sys_monitor_cpu_5_minutes
+  - MIB: TPLINK-SYSMONITOR-MIB
+    table:
+      name: tpSysMonitorMemoryTable
+      OID: 1.3.6.1.4.1.11863.6.4.1.2.1
+    symbols:
+      - name: memory.usage
+        OID: 1.3.6.1.4.1.11863.6.4.1.2.1.1.2
+    metric_tags:
+      - column:
+          OID: 1.3.6.1.4.1.11863.6.4.1.2.1.1.1
+          name: tpSysMonitorMemoryUnitNumber
+        tag: tp_sys_monitor_memory_unit_number
```

Comment:
There were no `metric_tags` and I added them myself. Link to the [MIB](http://www.circitor.fr/Mibs/Html/T/TPLINK-SYSMONITOR-MIB.php). Have I done it correctly?